src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
+        Colors figureColor = Colors.values()[index];
+        return figureColor.toString();
```

Comment:
Creating a new instance of Random every time the method is called can be inefficient. Consider creating a single instance of Random as a private final class member and reusing it for each method call.

Reply:
Creating a new instance of Random every time the method is called can be inefficient. Consider creating a single instance of Random as a private final class member and reusing it for each method call.